i18n-scripts/README.md

```diff
@@ -0,0 +1,55 @@
+# i18n-scripts
+
+The i18n-scripts folder contains all homebrewed scripts for i18n workflow automation
+in OpenShift.
+
+## Languages
+The list of languages we're using are stored in `languages.sh`. All scripts
+that rely on language information import this variable, and it should be updated
+whenever OpenShift gains support for an additional language.
+
+## Memsource Automation
+We have created two scripts to handle pushing and pulling translations to/from
+Memsource, the tool the Red Hat Globalization team uses for translation jobs.
+
+Before running either tool, you must first install the [unofficial Memsource CLI client](https://github.com/unofficial-memsource/memsource-cli-client#pip-install).
+You also have to [configure it with your Memsource login info](https://github.com/unofficial-memsource/memsource-cli-client#configuration-red-hat-enterprise-linux-derivatives).
+
+Once your login info is configured, you should be able to log in by running `source ~/.memsourcerc`.
+
+Example CLI usage for upload script: `yarn memsource-upload -v 4.8 -s 200`
+* -v is the current OpenShift version
+* -s is the current sprint number
+
+Example CLI usage for download script: `yarn memsource-download -p 6sB6qwpbRkGCeBQq4hUyK1`
+* -p is the project ID in Memsource. The project ID can be obtained from the Memsource project URL (it's the series of numbers and letters after /show/, i.e. https://cloud.memsource.com/web/project2/show/FBfZeTEWPYaC4VXhgrW0R2).
+
+## Build
+`build-i18n.sh` runs as part of the general `yarn i18n` command workflow. This script
+runs the i18next parser across the entire codebase and generates json files based on
+the marked keys in the application.
+
+## Folder Consolidation
+`consolidate-public-folders.js` runs as part of the general `yarn i18n` command workflow.
```

Comment:
I wouldn't think you'd need this script in your repo.

Reply:
Thanks, removed